internal/services/compute/windows_virtual_machine_scale_set_other_resource_test.go

```diff
@@ -537,12 +537,19 @@ func TestAccWindowsVirtualMachineScaleSet_updateHealthProbe(t *testing.T) {
 				check.That(data.ResourceName).ExistsInAzure(r),
 			),
 		},
+		data.ImportStep("admin_password"),
+		// Have to remove VMSS to change the sku of load balancer
```

Comment:
We should try and figure out how to not require this, ie we shouldn't require the user to apply multiple times to do this

Reply:
I've updated the test to create two sets of Load Balancer and other dependencies, thus removing the step to remove VMSS.
As the `sku` in Load Balancer is `ForceNew`, we are always moving into a new one when updating it in the VMSS, so it shall be fine to replicate a new one for the update scenario.